### ISSUE TITLE
Would be useful if command node output had a property that showed what command was executed

### ISSUE THREAD

**Issue by Botched1** [fblackburn1/node-red-contrib-hubitat#89]
One of the ideas I've thought about for a while was to make a subflow that verified a device state changed to the expected value after a command node command was issued.

It would be much easier to do post-processing after a command node if the command that the command node executed was in the output.

Potentially something like:
msg.command_ran
msg.arguments_used

Any thoughts on the value of adding something like that to the output msg from the command mode?

**Comment by fblackburn1** [comment]
Agree
The current output is the Maker API output (in `msg.response`) which is the device state before executing command ...

Do you think of any consequence to use the same name than input?
```
msg.command
msg.arguments
```

**Comment by Botched1** [comment]
Sorry I missed this... I got the notification email - but must have been busy, and then forgot about it!

The issue re-using msg.command and msg.arguments would be when users are doing multiple commands in serial. They would have to be sure to overwrite msg.command/msg.arguments, or delete them if they are specifying the command in the downstream command node. 

Would probably be better in a different variable(s).

**Comment by fblackburn1** [comment]
I don't forget you request, I'll have more time this weekend to check it :smiley:

**Comment by Botched1** [comment]
Because sometimes commands do not work (maker api issue, mesh congestion, etc) I have made a few subflows that check device status after a command is issued, and re-issues the command if the device status doesn't match the commanded action (I only support a small subset of commands - on/off/setLevel/a few custom device specific commands). 

In those subflows I do end up using msg.command and msg.arguments (where appropriate).

So I guess I changed my opinion on this. I think it would be OK just storing the command and arguments in msg.command and msg.arguments in all cases.
